include/pgduckdb/vendor/pg_fwd.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+// Postgres forward declarations
```

Comment:
Could use a bit more explanation why we have this file, and when we should add new things to it.